autoapi/mappers/dotnet.py

```diff
@@ -180,16 +180,16 @@ def output_rst(self, root, source_suffix):
                 filename = obj.name.split('(')[0]
             except IndexError:
                 filename = id
-            filename = filename.replace('#', '-')
+            filename = filename.replace('#', '-').replace('<', '-').replace('>', '')
```

Comment:
We should probably just be properly slugifying these. I made the < turn into a -, but killing the trailing >, but we should standardize on a schema here.